### DISCUSSION TITLE
Start date to trigger jobs

### DISCUSSION THREAD

**Discussion by pipeline1987** [dkron-io/dkron#1484] [Q&A]
Hello team, just as there is there a parameter to configure the `ExpiresAt` time, there is any option to indicate the start date of the job execution?

**Comment by NAlexandrov** [comment]
I added a new `StartedAt` field in a `Job` type.  Review is needed https://github.com/NAlexandrov/dkron/commit/be467c2fe109026491d6969e1b330c8b59431a72
